tests/functional/labels.js

```diff
@@ -0,0 +1,51 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+define([
+  'intern',
+  'intern!object',
+  'intern/chai!assert',
+  'require'
+], function (intern, registerSuite, assert, require) {
+  'use strict';
+
+  var url = function (num) {
+    return intern.config.siteRoot + '/issues/' + num;
+  };
+
+  registerSuite({
+    name: 'labels',
+
+    'label gear is visible': function () {
+      return this.remote
+        .setFindTimeout(intern.config.wc.pageLoadTimeout)
+        .get(require.toUrl(url(100)))
+        .sleep(10000)
+        .findByCssSelector('.LabelEditor-launcher span.wc-sronly')
+        .getVisibleText()
```

Comment:
Ah! misunderstood this feature. I didn't find a getText. :(

ok I see https://theintern.github.io/leadfoot/Command.html#isDisplayed
and https://theintern.github.io/leadfoot/Command.html#findDisplayedByCssSelector

but then what do you assert?

Do you know a way to have a kind of output on the terminal for understanding what is sent by this code. Instead of relying on black magic (for me. Tests/Failures is not very effective.)